ambari-infra-manager-it/pom.xml

```diff
@@ -182,6 +211,53 @@
         <directory>src/test/resources</directory>
       </testResource>
     </testResources>
+    <plugins>
+      <plugin>
+        <groupId>io.swagger</groupId>
+        <artifactId>swagger-codegen-maven-plugin</artifactId>
+        <version>2.3.1</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <inputSpec>${project.parent.basedir}/ambari-infra-manager/src/main/resources/swagger/swagger.yaml</inputSpec>
+              <language>java</language>
+              <generateApiTests>false</generateApiTests>
+              <generateModelTests>false</generateModelTests>
+              <output>${generated-sources-path}</output>
+              <configOptions>
+                <sourceFolder>${generated-sources-java-path}</sourceFolder>
+                <apiPackage>org.apache.ambari.infra.client.api</apiPackage>
+                <modelPackage>org.apache.ambari.infra.client.model</modelPackage>
+                <invokerPackage>org.apache.ambari.infra.client.invoker</invokerPackage>
+                <dateLibrary>java8</dateLibrary>
```

Comment:
is it will work with java11 or is it not depends on anything like that ?

Reply:
I think this affects only that the generated source is going to use java `java.time`.
I am going to prepare and test Infra manager for java11 in another PR.

Reply:
I've just run the integration tests with java11 and it works after adding `javax.annotation:javax.annotation-api` and `javax.xml.bind:jaxb-api`